collections/kapoorlabs.yml

```diff
@@ -0,0 +1,51 @@
+# each Collection must have a title
+title: KapoorLabs
+
+# the date the Collection was submitted or last updated. The date must be in ISO-8601 format within double quotes (e.g. "2022-05-24")
+updated_date: "2023-05-01"
+
+# a one-sentence summary of the collection. 
+# give napari users insight into what the collection contains and how it will help them.
+# this will be shown on the Collection's card.
+summary: Plugins created by developers at KapoorLabs, France
+
+
+# a longer description of the collection. 
+# give more background and context on the collection, what the plugins do, and why it matters.
+# this will be displayed at the top of the collection. See https://yaml-multiline.info/ for YAML syntax for multiline values.
+description: >
+  This is a collection of plugins developed at KapoorLabs, Paris. We develop several workflows for single cell biologists based on VollSeg Napari plugin and other python libraries created by us.
+
+# provide a cover image for the collection. 
+# this must be at least 1400px and be present in the top-level `images` folder in this repo (note: include full file name with the file extension)
+cover_image: kapoorlogo.png
+
+# the list of plugins in this collection
+plugins:
+    # the name of the plugin is the pypi name (e.g. napari-sim-processor)
+  - name: vollseg-napari
+    # provide a comment to let users know why this plugin matters
+    comment: Cell, Nuclei, Cytoplasm and filament kymograph segmentation and denoising. 
+  #copy the below template for all plugins in your collection
+  - name: vollseg-napari-mtrack
+    comment: Dynamic instability quantification of microtubule and actin filaments using VollSeg and Ransac.
+  
+  - name: vollseg-napari-trackmate
+    comment: Track and track statistics analysis using Fiji plugin TrackMate outputs and tracks layer in Napari.
+   
+# who created this collection? add your name, credentials, and relevant links
+curator:
+  name: Varun Kapoor
+  title: Data Scientist
+  affiliation:
+    institution: KapoorLabs, Paris
+    website: https://www.kapoorlabs.org
+  links:
+    # an ORCID ID is required
+    orcid: 0000-0001-5331-7966
+    # (optional) link to Twitter
+    twitter: https://twitter.com/kapoorlabs
+    # (optional) link to GitHub
+    github: https://github.com/Kapoorlabs-CAPED
+    # (optional) link to another website, such as a personal or institutional homepage
+    website: 

```

Comment:
Another option is to remove this null `website` field entirely so that we don't have to wait for Thursday's release to get this in.